src/main/java/duke/DukeIllegalCommandException.java

```diff
@@ -0,0 +1,11 @@
+package duke;
+
+/** Exception for unrecognised commands */
```

Comment:
Same issue regarding single line comments as above. Perhaps check through the rest of your classes for this issue as well as the issue regarding the periods at the end of lines?